assets/out

```diff
@@ -92,6 +98,13 @@ push_tags() {
   git push --tags push-target $forceflag
 }
 
+add_and_push_notes() {
+  if [ -n "$notes" ]; then
+    git notes add -m "${notes}"
```

Comment:
Done as you suggested @vito - agree it feels better.

How is this PR looking now?